zerver/models.py

```diff
@@ -1053,6 +1054,27 @@ def parse_usermessage_flags(val):
         mask <<= 1
     return flags
 
+class Attachment(models.Model):
+    MAX_FILENAME_LENGTH = 100
+    file_name = models.CharField(max_length=MAX_FILENAME_LENGTH, db_index=True)
+    path_id = models.TextField(default='/some/path')
```

Comment:
I intended path_id to be a storage location agnostic representation of the path of the file. If the path of a file is `http://localhost:9991/user_uploads/1/31/4CBjtTLYZhk66pZrF8hnYGwc/temp_file.py` then its path_id will be `1/31/4CBjtTLYZhk66pZrF8hnYGwc/temp_file.py`. I really think it will come in handy later